src/xrpld/app/tx/detail/DeleteAccount.cpp

```diff
@@ -203,7 +239,52 @@ DeleteAccount::preclaim(PreclaimContext const& ctx)
         return tecDST_TAG_NEEDED;
 
     // Check whether the destination account requires deposit authorization.
-    if (ctx.view.rules().enabled(featureDepositAuth) &&
+    if (ctx.tx.isFieldPresent(sfCredentialIDs))
```

Comment:
Shouldn't these checks only be done after Deposit Auth has been checked?

Reply:
No, cause depositAuth without credentials can exists (old way)